consensus/src/quorum_store/batch_generator.rs

```diff
@@ -277,6 +251,9 @@ impl BatchGenerator {
                     self.back_pressure = updated_back_pressure;
                 },
                 _ = interval.tick() => {
+                    // expire based on timestamp
+                    self.expire();
```

Comment:
As discussed offline, bring back the old logic to make sure er do not re-propose the same tnxs while batches are in progress.